pkg/server/api_buildimage_test.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	elementalv1 "github.com/rancher/elemental-operator/api/v1beta1"
+	"gotest.tools/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+func TestApiBuildImage(t *testing.T) {
```

Comment:
what do you think about using ginkgo/gomega here? this way stay aligned with controllers tests.

Reply:
Yes I agree, having all tests using the same framework is helpful for maintenance, it shouldn't be a big deal using ginkgo and gomega.

Reply:
agreed

Reply:
going to merge this, will do soon in a following PR